res/css/structures/_SpacePanel.pcss

```diff
@@ -390,8 +390,17 @@ limitations under the License.
     .mx_UserMenu {
         padding-bottom: 12px;
         border-bottom: 1px solid $separator;
-        margin: 12px 14px 4px 16px;
+        margin: 12px 14px 4px 18px;
+        width: min-content;
         max-width: 226px;
+
+        /* Display the container and img ehere as block elements so they don't take
+         * up extra vertical space.
+         */
+        span,
+        img {
```

Comment:
Could you target a class name here so we're not relying on the specific elements used in the Compound Avatar implementation?

Reply:
I can for the span, but the image only has an autogenerated class name. Unsure whether adding one just for this or just using the element type is the lesser evil. We could also match everything, which might actually be better?

Reply:
From my brief testing it looked like setting `display: block` only on the span was enough to fix the height, for what it's worth

Reply:
Huh, I retested and you're absolutely right.